security/seccomp/types.go

```diff
@@ -0,0 +1,20 @@
+package seccomp
+
+type SyscallCondition struct {
+	Argument uint   `json:"argument"`
+	Operator string `json:"operator"`
+	ValueOne uint64 `json:"value_one"`
+	ValueTwo uint64 `json:"value_two,omitempty"`
+}
+
+type BlockedSyscall struct {
+	Name       string             `json:"name,"`
+	Conditions []SyscallCondition `json:"conditions,omitempty"`
+}
+
+type SeccompConfig struct {
```

Comment:
I prefer call this type `Config` because `seccomp` part always obvious from package name.